M/mousetrap/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "mousetrap"
+version = v"0.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Clemapfel/mousetrap.git", "ffa28d0bd569118320a6bb063286edfb35fc0429"),
+    GitSource("https://github.com/Clemapfel/mousetrap_julia_binding.git", "6b838ea238e118d694ffc1b3a1e0225441c8cfb3"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+echo -e "[binaries]\ncmake='/usr/bin/cmake'" >> cmake_toolchain_patch.ini
+cd mousetrap
+mkdir ${prefix}/share/licenses/mousetrap
+cp LICENSE ${prefix}/share/licenses/mousetrap/LICENSE
+meson setup build --cross-file=$MESON_TARGET_TOOLCHAIN --cross-file=../cmake_toolchain_patch.ini
+meson install -C build
+cd ../mousetrap_julia_binding
+meson setup build --cross-file=$MESON_TARGET_TOOLCHAIN --cross-file=../cmake_toolchain_patch.ini -DJulia_INCLUDE_DIRS=$prefix/include/julia
+meson install -C build
+cd ..
+rm cmake_toolchain_patch.ini
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = Platform[]
+include("../../L/libjulia/common.jl")
+for version in [v"1.7.0", v"1.8.2", v"1.9.0", v"1.10", v"1.11"]
+    for platform in libjulia_platforms(version)
+        if !(arch(platform) == "i686" || contains(arch(platform), "arm"))
+            push!(platforms, platform)
+        end
+    end
```

Comment:
Are you just trying to not use any 32-bit platforms? (I note that this filter will leave only the 64-bit platforms).

Reply:
yes, exactly. I couldn't find a is64bit function so I did it with pattern matching

Reply:
You can use `nbits(platform)` to find out the platform, so to only get the 64-bit platforms you can use the comparison `nbits(platform) == 64`.

Reply:
that is much nicer, fixed by https://github.com/JuliaPackaging/Yggdrasil/pull/7461/commits/e6ef38bb1477de1a55f5e14193a74ffe091d61f1